opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -432,6 +432,12 @@ message IntHistogramDataPoint {
   // buckets if a histogram is provided.
   sfixed64 sum = 5;
 
+  // A histogram may optionally contain the distribution of the values in the
+  // population. In that case one the "explicit_bounds" and "bucket_counts"
+  // fields must be defined. Otherwise both fields must be omitted in which case
+  // the distribution of values in the histogram is unknown and only the total
+  // count and sum are known.
```

Comment:
Nit: This is a bit hard to parse.

Proposal:
```
If the histogram contains a distribution of values then one of the "explicit_bounds" or 
"bucket counts" fields must be defined.   
If the histogram is not a distribution of values, then both "explicit_bounds" and "bucket_counts"
must be omitted and only "count" and "sum" are known.
```

Reply:
Realized that the message was also wrong, we need both.